pkg/proxy/winuserspace/proxysocket.go

```diff
@@ -510,10 +512,9 @@ func (udp *udpProxySocket) ProxyLoop(service ServicePortPortalName, myInfo *serv
 			dnsSearch = append(dnsSearch, suffixList...)
 		}
 	}
-
+	klog.V(3).Infof("ProxyLoop is now starting for service %v", myInfo)
```

Comment:
This seems like it could be pretty noisy, even for v=3

Reply:
... hmmmm...  here was my thinking - but i wrote this PR like a month ago so i mighta forgot something --- lmk what im missing here -- 

- The goal is to make the userspace kube proxy logs `literate`, i.e. to tell the story around whats happening by default, given the complexity of managing services in user space .....    

- the default logging would not be modified by this - as you implied... so  at `v=3`,  you get  a new  log statement anyways  `for every new  service` iirc already which occurs in kube proxy .... So we're not algorithmically increasingly the amount of noise.  Also remember this code base is much more dangerous and difficult to reason about then the hardened linux kube proxy, so it could save us lots of time in production scenarios 

- also remember, every cni has bespoke kube proxy instructions, so this noise is less trivial then you might initially think?